sql/2019/18_Page_Weight/18_03.sql

```diff
@@ -0,0 +1,15 @@
+#standardSQL
+# 18_03: Distribution of response size by response format
+SELECT
+  percentile,
```

Comment:
Just noticed this one should also be selecting `_TABLE_SUFFIX AS client` and `GROUP BY client`